app_test.go

```diff
@@ -473,3 +491,77 @@ func TestNopLogger(t *testing.T) {
 	app := fxtest.New(t, NopLogger)
 	app.RequireStart().RequireStop()
 }
+
+type testWrappedErr struct {
+	graph string
+}
+
```

Comment:
We don't need to support custom implementations of wrappedError IMO

Reply:
Actually, it looks like testing is easier like this. In that case, can we rename the interface to `errorWithGraph` or something similar. "Wrapped" is too generic.

Also, in that case, we should maybe rename `Graph()` to something more verbose to avoid conflicts. Alternatively, it can return a `type GraphvizGraph string` instead of `string`, which guarantees that there are no conflicts with any other `error` implementation that happens to have a `Graph() string` method.

Reply:
I created `type DotGraph string` and changed the function to be `func Graph() DotGraph`